### DISCUSSION TITLE
Push to packages through actions bot: Audit log shows another completey unrelated user

### DISCUSSION THREAD

**Discussion by Siedlerchr** [community/community#26807] [Packages]
<p>Hi,</p>
<p>I noticed that  the audit log in my private organization repo shows info about a completely unrelated user:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/d/3d2dc1b16071a1a4ab07884646c919f0128e9efa.png" data-download-href="/uploads/short-url/8Jd9QnFOCRFSEzWAgUXBwCyhI82.png?dl=1" title="Bildschirmfoto 2021-06-24 um 08.43.09" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/320228/181099976-e9ea216b-982c-46db-8179-2e2657084947.png" alt="Bildschirmfoto 2021-06-24 um 08.43.09" data-base62-sha1="8Jd9QnFOCRFSEzWAgUXBwCyhI82" width="690" height="288" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3d2dc1b16071a1a4ab07884646c919f0128e9efa_2_690x288.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3d2dc1b16071a1a4ab07884646c919f0128e9efa_2_1035x432.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3d2dc1b16071a1a4ab07884646c919f0128e9efa_2_1380x576.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3d2dc1b16071a1a4ab07884646c919f0128e9efa_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Bildschirmfoto 2021-06-24 um 08.43.09</span><span class="informations">1670×698 90.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>When I click on the profile it takes me to <a href="https://github.com/kelleyduran" class="inline-onebox" rel="noopener nofollow ugc">kelleyduran (Kelley Duran) · GitHub</a> a user that I don’t know and is not associated in any way with our org.</p>
<p>Anyone has experienced the same? This looks dubios to me</p>

**Comment by shayneburgess** [comment]
<p>This is a bug in the UI for the audit log. We are working on a fix to show the correct user picture.</p>

**Comment by Siedlerchr** [comment]
<p>Thought so as well. Thanks for the info</p>

**Comment by shayneburgess** [comment, marked as answer]
<p>Just to follow up, this is fixed for new Audit log entries moving forward.</p>

**Comment by Siedlerchr** [comment]
<p>Thanks for the follow up°</p>
